src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <div className="cont-button">
+      <div className="btn-row">
+        <Button name="Ac" />
```

Comment:
Let's pass some color to the buttons that are not going to default to orange, for example:

```jsx
<Button name="Ac" color='#e0e0e0' />
```